supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalBackend.scala

```diff
@@ -1,34 +1,59 @@
 package cromwell.backend.impl.local
 
-import cromwell.backend.BackendJobExecutionActor.BackendJobExecutionResponse
+import java.nio.file.FileSystems
+
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionFailedResponse, BackendJobExecutionResponse}
 import cromwell.backend.BackendLifecycleActor.JobAbortResponse
 import cromwell.backend._
+import org.slf4j.LoggerFactory
 import wdl4s.Call
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scala.util.Try
 
-class LocalBackend extends BackendJobExecutionActor {
-  /**
-    * Restart or resume a previously-started job.
-    */
-  override def recover(jobDescriptor: BackendJobDescriptor): Future[BackendJobExecutionResponse] = ???
+object LocalBackend {
+  val logger = LoggerFactory.getLogger("LocalBackend")
+}
 
-  /**
-    * Execute a new job.
-    */
-  override def execute(jobDescriptor: BackendJobDescriptor): Future[BackendJobExecutionResponse] = ???
+class LocalBackend(override val workflowDescriptor: BackendWorkflowDescriptor,
+                   override val configurationDescriptor: BackendConfigurationDescriptor,
+                   override val calls: Seq[Call]) extends BackendJobExecutionActor {
 
-  override protected def workflowDescriptor: BackendWorkflowDescriptor = ???
+  override protected implicit def ec: ExecutionContext = scala.concurrent.ExecutionContext.global
 
-  override protected def configurationDescriptor: BackendConfigurationDescriptor = ???
+  // FIXME Boilerplate ? Move to Backend trait ?
+  val config = configurationDescriptor.config.getConfig(configurationDescriptor.configPath)
+  // TODO Support GCS ?
+  val fileSystems = List(FileSystems.getDefault)
+  val workflowPaths = new WorkflowPaths(workflowDescriptor, config, fileSystems)
+
+  // TODO Do we need to evaluate runtime attributes at the workflow level ?
+
+  override def preStart() = {
+    workflowPaths.workflowRoot.toFile.mkdirs()
+  }
 
   /**
-    * The set of calls which this backend will be expected to run
+    * Execute a new job.
     */
-  override protected def calls: Seq[Call] = ???
+  override def execute(jobDescriptor: BackendJobDescriptor): Future[BackendJobExecutionResponse] = {
+    Try(
+      new LocalExecutor(workflowDescriptor, jobDescriptor, fileSystems, config)
+    ) map { _.execute } recover {
+      case e => Future.successful(BackendJobExecutionFailedResponse(jobDescriptor.key,
+        new RuntimeException(s"Failed to initialize job ${jobDescriptor.key}", e))
+      )
+    } get
```

Comment:
I can't quite tell yet but it seems like callers of `execute` are not equipped to receive an exception if the `Try`.  Very possible I'm missing something though